code-challenges/401challenges/src/main/java/challenges/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,72 @@
+package challenges.linkedlist;
+
+import java.util.ArrayList;
+
+public class LinkedList {
+   public  Node head;
+   public int length;
+
+    public void LinkedList(){
+            this.head = null;
+            this.length = 0;
+    }
+    public void insert(int data){
+        try
+        {
+            Node node = new Node(data);
+            node.next = null;
+            if(this.head == null){
+                this.head = node;
+                this.length++;
+            } else {
+                Node current = this.head;
```

Comment:
If line 23 was `node.next = this.head` then you could get rid of this line completely